server_admin/topics/threat/brute-force.adoc

```diff
@@ -1,84 +1,102 @@
+|The minimum time between login attempts.
+|1000 milliseconds.
+
+|Minimum Quick Login Wait
+|The minimum time the user is disabled when login attempts are quicker than _Quick Login Check Milliseconds_.
+|1 minute.
+
+|===
+
+*Permanent Lockout Flow*
 ====
 . On successful login
 .. Reset `count`
 . On failed login
 .. Increment `count`
 .. If `count` greater than _Max Login Failures_
 ... Permanently disable user
-.. Else if time between this failure and the last failure is less than _Quick Login Check Milli Seconds_
+.. Else if the time between this failure and the last failure is less than _Quick Login Check Milliseconds_
 ... Temporarily disable user for _Minimum Quick Login Wait_
 
-When a user is disabled they can not login until an administrator enables the user; enabling an account resets `count`.
+When {project_name} disables a user, the user cannot log in until an administrator enables the user. Enabling an account resets the `count`.
 ====
 
+*Temporary Lockout Parameters*
+
+|===
+|Name |Description |Default
+
+|Wait Increment
+|The time added to the time a user is temporarily disabled when the user's login attempts exceed _Max Login Failures_.
+|1 minute.
+
+|Max Wait
+|The maximum time a user is temporarily disabled.
+|15 minutes.
+
+|Failure Reset Time
+|The time when the failure count resets. The timer runs from the last failed login.
+|12 hours.
+
+|===
+
 *Temporary Lockout Algorithm*
 ====
 . On successful login
 .. Reset `count`
 . On failed login
-.. If time between this failure and the last failure is greater than _Failure Reset Time_
+.. If the time between this failure and the last failure is greater than _Failure Reset Time_
 ... Reset `count`
 .. Increment `count`
-.. Calculate `wait` using _Wait Increment_ * (`count` / _Max Login Failures_). The division is an integer division so will always be rounded down to a whole number
-.. If `wait` equals 0 and time between this failure and the last failure is less than _Quick Login Check Milli Seconds_ then set `wait` to _Minimum Quick Login Wait_ instead
+.. Calculate `wait` using _Wait Increment_ * (`count` / _Max Login Failures_). The division is an integer division rounded down to a whole number
+.. If `wait` equals 0 and the time between this failure and the last failure is less than _Quick Login Check Milliseconds_, set `wait` to _Minimum Quick Login Wait_.
 ... Temporarily disable the user for the smaller of `wait` and _Max Wait_ seconds
 
-Login failures when a user is temporarily disabled do not increment `count`.
+'count` does not increment when a temporarily disabled account commits a login failure.
 ====
 
-The downside of {project_name} brute force detection is that the server becomes vulnerable to denial of service attacks.
-An attacker can simply try to guess passwords for any accounts it knows and these account will be disabled.
-Eventually we will expand this functionality to take client IP address into account when deciding whether to block a user.
```

Comment:
Is this sentence elsewhere or is it removed because we should not promise future functionality?

Reply:
Yes, I removed this because we should not promise future functionality.